src/client/Microsoft.Identity.Client/Core/Constants.cs

```diff
@@ -13,7 +13,7 @@ internal static class Constants
         public const int CodeVerifierByteSize = 32;
 
         public const string UapWEBRedirectUri = "https://sso"; // only ADAL supports WEB
-        public const string DefaultRedirectUri = "urn:ietf:wg:oauth:2.0:oob";
+        public const string DefaultRedirectUri = "msal4a1aa1d5-c567-49d0-ad0b-cd957a47f842://auth"; // in msidentity-samples-testing tenant -> PublicClientSample
```

Comment:
Thanks for adding the information about the tenant. This will be easier to maintain in the future.